config/database.yml.example

```diff
@@ -3,15 +3,17 @@
 #
 #   Ensure the SQLite 3 gem is defined in your Gemfile
 #   gem 'sqlite3'
+
 development:
   adapter: sqlite3
   database: db/development.sqlite3
   pool: 5
   timeout: 5000
 
-# Warning: The database defined as "test" will be erased and
-# re-generated from your development database when you run "rake".
-# Do not set this db to the same as development or production.
+#   Warning: The database defined as "test" will be erased and
```

Comment:
Why did you make this change? 
Using spaces, instead of a tab, is fine. Please revert this back.

Reply:
Please use spaces.